articles/variables-in-js/index.md

```diff
@@ -0,0 +1,129 @@
+# Understanding Variables In Javascript
+
+## Introduction
+Variables are a way for programmers to name a value to reuse it, update it, or keep track of it.
+In this article, we will go through naming variables, declaring them, and initializing them.
+We are also going to touch on the scope of variables to ensure that we understand variables fully.
```

Comment:
you can use `completely` instead of `fully`

Reply:
Rewrite as

We are also going to touch on the scope of variables to ensure that we completely understand variables.